### DISCUSSION TITLE
Suggest existing token names when using aliases

### DISCUSSION THREAD

**Discussion by renderghost** [six7/figma-tokens#643] [Ideas]
Currently, when I'm creating new tokens using aliases, I need to remember the exact names of existing tokens. this gets very complex and getting it wrong can have big implications.

![image](https://user-images.githubusercontent.com/21337875/165477173-c7221546-ebc8-4358-894e-ca8b8dabd73b.png)

Suggesting existing tokens, as I type, would be so helpful.

![image](https://user-images.githubusercontent.com/21337875/165478083-5c21eb3c-31f6-40fe-960b-579bbdff61eb.png)

Even better, show the group name to help me make a perfect match

![image](https://user-images.githubusercontent.com/21337875/165478531-5811ba1b-648d-4b48-9114-4cf0117d492f.png)

(ignore the nonsensical names I've used here)

Thanks so much!

**Comment by six7** [comment]
This is going to be part of the next release! It will allow you to pick a token, autocomplete a token on typing { and use that to search for a token. 🎉

**Reply by renderghost** [reply]
That's really exciting to hear! Thanks 👍

**Comment by six7** [comment]
This is now released for basic token values!

**Comment by bosy385** [comment]
That's an awesome improvement, but would it be possible to have a similar way of accessing reference/alias tokens on the Inspect tab ? 
My use case would be to check tokens for an element selected in Figma from the inspect tab and then e.g. change the fill color by selecting another valid color token from there. Currently it's a text input field (with no error checking or autocompletion).
<img width="429" alt="image" src="https://user-images.githubusercontent.com/105915323/170300571-925c58ae-00b8-42c6-b2ec-7a74f6a61265.png">
That way I could change the look and feel (size, color, radii, ..)  of a component in **one place**

**Reply by six7** [reply]
This is coming up in the next release! 🎉
